tests/unit/plugins/filter/test_ipaddr.py

```diff
@@ -895,3 +895,7 @@ def test_hwaddr(self):
     def test_ip4_hex(self):
         self.assertEqual(ipaddr.ip4_hex("192.0.2.24"), "c0000218")
         self.assertEqual(ipaddr.ip4_hex("192.0.2.24", "."), "c0.00.02.18")
+
+    def test_hex_ip4(self):
+        self.assertEqual(ipaddr.ip4_hex("c0000218"), "192.0.2.24")
+        self.assertEqual(ipaddr.ip4_hex("c0000218", ":"), "192:0:2:24")
```

Comment:
I'm not sure if this is a useful representation of an ipv4 address... what you really want with `delimiter`, I would imagine, is symmetry with `ip4_hex`, that is the ability to parse "c0.00.02.18" or "c0:00:02:18" in addition to "c0000218".